python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -85,8 +85,8 @@ def __str__(self):
 
     def append(self, value):
         self.addProcess(value)
-        self.process_objects = sorted(
-            self.process_objects, key=lambda process: process.order)
+        if hasattr(self.process_objects, 'order'):
+            self.process_objects = sorted(self.process_objects, key=lambda process: process.order)
```

Comment:
So this looks like it's probably a proxy-layer violation? Somehow, proxied objects are being passed to this function.